src/recruiter/recruiter.service.ts

```diff
@@ -44,17 +48,100 @@ export class RecruiterService {
     const oneRecruiter = await this.dataSource
       .getRepository(Recruiter)
       .createQueryBuilder('recruiter')
-      .where('recruiter.id = :id', {id})
-      .andWhere('recruiter.registerToken = :registerToken', {registerToken})
+      .where('recruiter.id = :id', { id })
+      .andWhere('recruiter.registerToken = :registerToken', { registerToken })
       .getOne();
 
-      return oneRecruiter ? {
-        isOk: true,
-        id: oneRecruiter.id,
-        role: 'Recruiter',
+    return oneRecruiter
+      ? {
+          isOk: true,
+          id: oneRecruiter.id,
+          role: 'Recruiter',
+        }
+      : {
+          isOk: false,
+        };
+  }
+
+  async changeStatus(id: string, status: string) {
+    //@TODO wysyłanie informacji do admina, ewentualna zmiana nazewnictwa w case'ach, lepsza walidacja błędów
+    //@TODO upewnienie się, że rekruter po wpisaniu w ściężkę id innego studenta niż ma w dodanych dostanie informację o braku dostępu
```

Comment:
tutaj moja mała podpowiedź jak można to zrobić:
każdy Student ma pole studentImport z id Recruiter, wystarczy tylko porównać id aktualnie zalogowanego Recruiter z polem studentImport w Student